components/sidebar/items/all/sidebar-display-item.tsx

```diff
@@ -73,15 +75,83 @@ export const SidebarItem: FC<SidebarItemProps> = ({
     }
   }
 
-  // const handleClickAction = async (
-  //   e: React.MouseEvent<SVGSVGElement, MouseEvent>
-  // ) => {
-  //   e.stopPropagation()
-
-  //   const action = actionMap[contentType]
-
-  //   await action(item as any)
-  // }
+  let iconComponent
+
+  const hoverClass = isHovering
+    ? "text-pixelspace-pink"
+    : "text-pixelspace-gray-40"
+
+  switch (contentType) {
+    case "prompts":
+      iconComponent = (
+        <i
+          className={`fa-solid fa-sparkle ${hoverClass}`}
+          style={{ fontSize: 12 }}
+        ></i>
+      )
+      break
+
+    case "files":
+      iconComponent = (
+        <i
+          className={`fa-regular fa-file ${hoverClass}`}
+          style={{ fontSize: 12 }}
+        ></i>
+      )
+      break
+    case "collections":
+      iconComponent = (
+        <i
+          className={`fa-regular fa-layer-group ${hoverClass}`}
+          style={{ fontSize: 12 }}
+        ></i>
+      )
+      break
+    case "assistants":
+      const assistant = item as any
+      iconComponent = (assistant.image_path as any) ? (
+        <Image
+          src={
+            assistantImages.find(image => image.path === assistant.image_path)
+              ?.url || ""
+          }
+          alt={item.name}
+          width={32}
+          height={32}
+          className="mt-2 size-5 cursor-pointer rounded-full hover:opacity-50"
+        />
+      ) : (
+        <div className="bg-pixelspace-gray-60 flex size-5 items-center justify-center rounded-full">
+          <i
+            className="fa-regular fa-robot text-pixelspace-gray-20"
+            style={{ fontSize: 11 }}
```

Comment:
este fontSize no podria tener un font standar en la app? No hay un bodySize of algo asi?